src/NuGet.Core/NuGet.Frameworks/FrameworkConstants.cs

```diff
@@ -35,11 +35,17 @@ public static class PlatformIdentifiers
         }
 
         /// <summary>
-        /// Allowed list of profiles in Net5.0ERA
+        /// Allowed list of platforms in Net5.0 Era
         /// </summary>
-        internal static HashSet<string> FrameworkProfiles
-            = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
-                { };
+        internal static class FrameworkPlatforms
+        {
+            public const string Android = "Android";
+            public const string IOS = "iOS";
+            public const string MacOS = "MacOS";
```

Comment:
The official name is `macOS` now:

```suggestion
            public const string MacOS = "macOS";
```